BuildResidentialHPXML/measure.rb

```diff
@@ -1423,7 +1423,7 @@ def arguments(model) # rubocop:disable Lint/UnusedMethodArgument
 
     arg = OpenStudio::Measure::OSArgument::makeChoiceArgument('heat_pump_cooling_compressor_type', compressor_type_choices, false)
```

Comment:
Now that it's required for ASHP, MSHP, and GSHP (the same applies to cooling systems), should we add error-checking if it's not provided for these system types?

Reply:
I think it's fine to just rely on the downstream schematron error-checking. But at a minimum, we should make sure the description is updated so the user knows which system types this argument is required for.